File: arduinobot_ws/src/arduinobot_test/script/simple_service.py
#!/usr/bin/env python3
import rospy

from arduinobot_test.srv import AddTwoInts, AddTwoIntsResponse

def add_two_int(req):
	rospy.loginfo(f"Ready to RUMBLEEEE: sum {req.a} and {req.b}")
	return AddTwoIntsResponse(req.a + req.b)


if __name__ == '__main__':
	rospy.init_node("simple_service")
	service = rospy.Service('add_two_ints', AddTwoInts, add_two_int)

	rospy.loginfo("The service is ready to RUMBLEEEE (just add two ints)")

	rospy.spin()

File: arduinobot_ws/src/arduinobot_test/src/simple_service.cpp
#include "ros/ros.h"
#include "arduinobot_test/AddTwoInts.h"

bool add(arduinobot_test::AddTwoInts::Request &req, arduinobot_test::AddTwoInts::Response &res)
{
	ROS_INFO("The service is ready to RUMBLEEEEE (just adding some ints :( %d and %d))", (int)req.a, (int)req.b);
	res.sum = req.a + req.b;

	return true;
}


int main(int argc, char **argv)
{
	ros::init(argc, argv, "simple_service");
	ros::NodeHandle n;
	ros::ServiceServer service = n.advertiseService("add_two_ints", add);

	ROS_INFO("The service is ready to RUMBLEEEEE (just adding some ints :( ))");

	ros::spin();

	return 0;
}